encord/objects/ontology_labels_impl.py

```diff
@@ -1626,6 +1629,22 @@ def _to_object_answers(self) -> Dict[str, Any]:
                 "classifications": list(reversed(all_static_answers)),
                 "objectHash": obj.object_hash,
             }
+
+            # At some point, we also want to add these to the other modalities
+            if self.data_type == DataType.AUDIO:
+                annotation = obj.get_annotations()[0]
+                ret[obj.object_hash]["range"] = [[range.start, range.end] for range in obj.range_list]
+                ret[obj.object_hash]["createdBy"] = annotation.created_by
+                ret[obj.object_hash]["createdAt"] = annotation.created_at.strftime(DATETIME_LONG_STRING_FORMAT)
+                ret[obj.object_hash]["lastEditedBy"] = annotation.last_edited_by
+                ret[obj.object_hash]["lastEditedAt"] = annotation.last_edited_at.strftime(DATETIME_LONG_STRING_FORMAT)
+                ret[obj.object_hash]["manualAnnotation"] = annotation.manual_annotation
```

Comment:
Is it possible to hide these values (i.e. created. lastEdited and manualAnnotation info) now? We want to store these values, but don't give the user the ability to export it and see it with `to_encord_dict`. The worry is that they might come to rely on these values, and it'll become harder for us to remove/change them.

I also appreciate this might be abit of a faff because saving labels uses this `to_encord_dict` method as well. And we do still want to update those values in the DB if changes are made via the SDK.

What do u think?

Reply:
would also need to hide get_annotation(0) to be unsupported as it can be accessed there as well

Reply:
the encord dict is undocumented anyway isn't it?

Reply:
discussing in slack